stories/wp-dashboard.stories.js

```diff
@@ -40,12 +40,14 @@ import WPAnalyticsDashboardWidgetTopPagesTable from '../assets/js/modules/analyt
 import { STORE_NAME as CORE_SITE } from '../assets/js/googlesitekit/datastore/site/constants';
 import { STORE_NAME as CORE_USER } from '../assets/js/googlesitekit/datastore/user/constants';
 import { WithTestRegistry } from '../tests/js/utils';
+import { disableFeature } from './utils/features';
 
 storiesOf( 'WordPress', module )
 	.add( 'WordPress Dashboard', () => {
 		global._googlesitekitLegacyData = wpDashboardData;
 		global._googlesitekitLegacyData.admin.assetsRoot = ''; // See .storybook/config.js
 		global._googlesitekitLegacyData.canAdsRun = true;
+		disableFeature( 'widgets.wpDashboard' );
```

Comment:
This call shouldn't be necessary as all features are to be disabled as part of the setup for each story (this is a more recent change so you probably haven't seen it yet).

Please add it here instead 🙂  https://github.com/google/site-kit-wp/blob/b9e590c436de83d1b198bcbf5ffc6c70b86f2c70/.storybook/config.js#L39-L45